src/EnergyPlus/ThermalComfort.cc

```diff
@@ -259,11 +259,15 @@ namespace ThermalComfort {
 	Array1D< ThermalComfortDataType > ThermalComfortData;
 	Array1D< AngleFactorData > AngleFactorList; // Angle Factor List data for each Angle Factor List
 
+	bool ASH55Flag( false );
+	Real64 runningAverageASH( 0.0 );
+
 	// Functions
 	void
 	clear_state()
 	{
 		FirstTimeFlag = true;
+		ASH55Flag = false;
```

Comment:
@mjwitte 
runningAverageASH is added in clear_state() in ThermalComfort.cc

I like your suggestion. Move ASH55Flag back and use DataHeatBalance::AdaptiveComfortRequested_ASH55.

Thanks.